docs/api.rst

```diff
@@ -259,6 +259,35 @@ Coalescent and approximations
 
 .. autoclass:: msprime.SmcPrimeApproxCoalescent
 
++++++++++++++++++++++++++++
+Multiple merger coalescents
++++++++++++++++++++++++++++
+
+Some evolutionary scenarios, such as a skewed offspring distribution
+combined with a type III survivorship curve, range expansion, and
+rapid adaptation, can predict diploid genealogies with up to four
+simultaneous multiple mergers. Msprime provides the option to simulate
+from two such genealogical processes.
+
+To use the Beta-Xi-coalescent, set the flag 
```

Comment:
I don't think the examples are terribly helpful here. Probably better to just point the reader to the tutorial and do all of the examples there. 
```
See :ref:`sec_tutorial_multiple_mergers` for examples of how to use these multiple merger coalescents.